## Replace the ... text below with a title and a summary of the problem.
## Feel free to remove any remaining comments once you're done!

= Asking for Help: I'm taking an unhandled threading exception that I've been unable to locate =

I keep getting this unhandled exception, but I believe I have placed try except statements in all my threads.  Any thoughts?

{{{
Unhandled exception in thread started by <bound method Thread.__bootstrap of <Thread(Thread-169204, stopped 1004)>>
Traceback (most recent call last):
  File "C:\Python27\lib\threading.py", line 503, in __bootstrap
    self.__bootstrap_inner()
  File "C:\Python27\lib\threading.py", line 518, in __bootstrap_inner
    del _limbo[self]
KeyError: <Thread(Thread-169204, stopped 1004)>
}}}

A philosophical discussion appears to be appropriate here.  How in fact does one get one's self out of _limbo? The answer my friend is blowing in the wind. The answer is ...
## Leave the note below so that editors can follow the instructions...

{{{#!wiki note
When ''answering'' questions, add the CategoryAskingForHelpAnswered category when saving the page. This will move the link to this page from the questions section to the answers section on the [[Asking for Help]] page.
}}}

----
CategoryAskingForHelp
